gemsfx/src/main/java/com/dlsc/gemsfx/SearchField.java

```diff
@@ -870,6 +877,39 @@ public final Node getPlaceholder() {
         return placeholder == null ? null : placeholder.get();
     }
 
+    private BooleanProperty autoSubmitOnBlur;
```

Comment:
"blur" is kind of the opposite of "focus" but nobody uses it in API. We also do not "submit" but "commit". So please change to `autoCommitOnFocusLost`.